docs/source/_static/release-notes.css

```diff
@@ -22,6 +22,14 @@
   color: white;
 }
 
+.changetype.changetype-logs {
+  background: #D3D62C;
+}
+
+.changetype.changetype-metrics {
+  background: #D3D62C;
```

Comment:
nit: I'd suggest using the Blueprint colours - http://blueprintjs.com/docs/#core/colors, perhaps `gold4` (`F2B824`) for consistency with some of the other change-types.